pyvista/core/dataobject.py

```diff
@@ -372,7 +372,11 @@ def add_field_data(self, array: np.ndarray, name: str, deep=True):
         >>> mesh.add_field_data(['I could', 'write', 'notes', 'here'],
         ...                      'my-field-data')
         >>> mesh['my-field-data']
-        pyvista_ndarray(['I could', 'write', 'notes', 'here'], dtype='<U7')
+        pyvista_ndarray([[['I could'],
+                          ['notes']],
+        <BLANKLINE>
+                         [['write'],
+                          ['here']]], dtype='<U7')
 
```

Comment:
If my understanding is correct, this is buggy now. Field data should not have any shape associated with the grid, so we should keep field data intact. I assume this means having to special-case field data in `DataSetAttributes`.